.teamcity/src/main/kotlin/promotion/PublishRelease.kt

```diff
@@ -30,7 +30,7 @@ abstract class PublishRelease(
     triggerName = "ReadyforRelease",
     gitUserEmail = "%gitUserEmail%",
     gitUserName = "%gitUserName%",
-    extraParameters = "-PconfirmationCode=%confirmationCode%"
+    extraParameters =""" "-PconfirmationCode=%confirmationCode%" "-PbotGradleGitHubToken=%github.bot-gradle.token%" """
```

Comment:
Does this cause it to be passed via the command line?

Reply:
You are right.